pontoon/base/static/js/l20n.js

```diff
@@ -0,0 +1,2831 @@
+var L20n = (function () {
+  'use strict';
+
+  /*eslint no-magic-numbers: [0]*/
+
+  const locales2rules = {
```

Comment:
I suppose that rules don't change often but We could manage a single source of them and generate this map from a database/different definition.